paasta_tools/utils.py

```diff
@@ -861,7 +861,7 @@ def check_deploy_group(self) -> Tuple[bool, str]:
             if deploy_group not in pipeline_deploy_groups:
                 return (
                     False,
-                    f"{self.service}.{self.instance} uses deploy_group {deploy_group}, but it is not deploy.yaml",
+                    f"{self.service}.{self.instance} uses deploy_group {deploy_group}, but {deploy_group} is not deployed to in deploy.yaml",
```

Comment:
Actually I missed this secondary typo: s/deployed to in/deployed in/ ?

This line is just cursed

Reply:
this line is indeed cursed - I think this is correct (we're telling users that we're not deploying TO deploy group IN deploy.yaml), but it still doesn't read/flow very well

maybe `but {deploy_group} is not referenced in deploy.yaml`?

Reply:
Ohhh i get it now, I can haz reading comprehension.  But agreed the second option is more obvious.